docs/gravity-bridge-setup-guide.md

```diff
@@ -0,0 +1,105 @@
+# Gravity Bridge Setup Guide
+
+## Prerequisite
+
+### Binaries
+
+- `geth`, the go-ethereum binary.
+- `cronosd`, the cronos node binary.
+- `orchestrator`, the gravity bridge orchestrator cli.
+- `pystarport`, a tool to run local cosmos devnet.
+- `start-geth`/`start-cronos`, convenient scripts to start the local devnets.
+
+Clone cronos repo locally and run `nix-shell integration_tests/shell.nix` in it, you'll get a virtual shell with the
+above essential binaries setup in `PATH`.
+
+### Ethereum Testnet
+
+You can either use a public testnet, or run `start-geth /tmp/test-geth` to get a local Ethereum testnet.
+
+You should own some funds in this testnet, for the local testnet, you can get the funds using this mnemonic words:
+`visit craft resemble online window solution west chuckle music diesel vital settle comic tribe project blame bulb armed
+flower region sausage mercy arrive release`.
+
+### Cronos Testnet
+
+You can either use a public cronos testnet (that have embed the gravity-bridge module), or run `start-cronos
+/tmp/test-cronos` to get a local Cronos testnet.
+
+You should own some funds in this testnet, for the local testnet, you'll get the funds with the same private key as
+above.
+
+## Generate Orchestrator Keys
+
+You need to prepare two accounts for the orchestrator, one for ethereum and one for cronos, you can also use a same
+private key for both accounts. You should transfer some funds to these accounts, so the orchestrator can cover the gas
+fees of messages relaying later.
+
+## Sign Validator Address
+
+To register the orchestrator with the validator, you need to sign a protobuf encoded message using the orchestrator's
+ethereum key, and send it to a cronos validator to register it.
+
+The protobuf message is like this:
+
+```protobuf
+message DelegateKeysSignMsg {
+  // The valoper prefixed cronos validator address
+  string validator_address = 1;
+  // Current nonce of the validator account
+  uint64 nonce = 2;
+}
+```
+
+Use your favorite protobuf library to encode the message, and use your favorite web3 library to do the messge signing,
+for example, this is how it could be done in python:
+
+```python
+msg = DelegateKeysSignMsg(validator_address=val_addr, nonce=nonce)
+sign_bytes = eth_utils.keccak(msg.SerializeToString())
+
+acct = eth_account.Account.from_key(...)
+signed = acct.sign_message(eth_account.messages.encode_defunct(sign_bytes))
+return eth_utils.to_hex(signed.signature)
```

Comment:
as a todo, create a script for this 🙏

Reply:
`gorc sign-delegate-keys` cando that, I've created an issue to migrate from orchestrator cli to gorc cli later.
https://github.com/crypto-org-chain/cronos/issues/78